runtime/strategies/create-views.js

```diff
@@ -17,18 +17,16 @@ export default class CreateViews extends Strategy {
       onView(recipe, view) {
         let counts = RecipeUtil.directionCounts(view);
 
-        let score = 1;
-        if (counts.in == 0 || counts.out == 0) {
-          if (counts.unknown > 0)
-            return;
-          if (counts.in == 0)
-            score = -1;
-          else
-            score = 0;
+        // Don't make a 'create' handle, unless there is someone reading,
+        // someone writing and at least 2 particles invloved.
+        if (counts.in == 0 || counts.out == 0
```

Comment:
Add tests for this?

Reply:
I was searching for tests for strategies and found none, only today Shane pointed out that they are in planner-tests.js. Added tests in a dedicated file this time and will move some of the other unit tests for strategies to their own files in follow up patches. Thanks for point it out and pushing me a bit.

Reply:
As an FYI, I'm in the process of adding a 'strategy-sequence-test.js' test file that lets you test sequences of strategies. It'll land with #1052

Reply:
Thanks! Sounds good. 👍